tagstory/src/main/java/no/tagstory/story/activity/StoryActivity.java

```diff
@@ -144,4 +157,43 @@ public void onClick(View view) {
 			}
 		});
 	}
+
+
+	@Override
+	public boolean onCreateOptionsMenu(Menu menu) {
+		getMenuInflater().inflate(R.menu.tag_menu, menu);
+		return true;
+	}
+
+	@Override
+	public boolean onMenuItemSelected(int featureId, MenuItem item) {
+		switch (item.getItemId()) {
+			case R.id.save_and_quit_story:
+				saveAndQuitStory();
+				break;
+		}
+		return true;
+	}
+
+	public void saveAndQuitStory() {
+		//TODO
```

Comment:
Fint hvis todo-meldingen står på samme linje, for da kan man lese den i todo-taben til IDEA